collect_app/src/test/java/org/odk/collect/android/support/FormUtils.java

```diff
@@ -28,12 +28,12 @@ public static String createXForm(String id, String version) {
                 "</h:html>";
     }
 
-    public static Form.Builder buildForm(long id, String jrFormId, String jrVersion, String formFilesPath) {
-        return buildForm(id, jrFormId, jrVersion, formFilesPath, "blah");
+    public static Form.Builder buildForm(long id, String formId, String version, String formFilesPath) {
+        return buildForm(id, formId, version, formFilesPath, "blah");
     }
 
-    public static Form.Builder buildForm(long id, String jrFormId, String jrVersion, String formFilesPath, String xform) {
-        String fileName = jrFormId + "-" + jrVersion;
+    public static Form.Builder buildForm(long id, String formId, String version, String formFilesPath, String xform) {
+        String fileName = formId + "-" + version + "-" + Math.random();
```

Comment:
This is so that we can represent the case where there are two forms with the same id and version.